lib/situation_room/sites.ex

```diff
@@ -1,8 +1,27 @@
 defmodule SituationRoom.Site do
+
   use Ecto.Schema
+  import Ecto.Changeset
 
   schema "sites" do
     field :endpoint, :string
     field :name, :string
   end
+
+  # @spec changeset(
```

Comment:
I'm not fully sure this spec is correct, but really you don't need to include a spec here if you don't want

Reply:
This spec popped out of nowhere at one point. I think it was autogenerated.